### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by basafilm**

### CODE REVIEW

File: javascript/javascript1/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+         return firstName +' ' + surName;
+    }
+};
+console.log(getFullName('', '', ''));
+
+
+//Event application
+
+function weekdayEvent(event) {
+let weekDays= ['Sunday','Monday','Tursday','Wedensday','Thursday', 'Friday', 'Sutarday'];
```
Comment (florinmaticu) [inline javascript/javascript1/week2/index.js]: THink of using `const` here
```suggestion
const weekDays= ['Sunday','Monday','Tursday','Wedensday','Thursday', 'Friday', 'Sutarday'];
```

File: javascript/javascript1/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+let today= new Date().getDay();
```
Comment (florinmaticu) [inline javascript/javascript1/week2/index.js]:
```suggestion
const today= new Date().getDay();
```

File: javascript/javascript1/week2/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+const eventDay = (today+event) % weekDays.length;
+return weekDays[eventDay];
+
+}
+console.log(weekdayEvent(6));
```
Comment (florinmaticu) [inline javascript/javascript1/week2/index.js]: what would happen if I do here `console.log(weekdayEvent(-8))`

File: javascript/javascript1/week2/index.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+function wearSomeThing(temprature){
+    
+    if(temprature>=18){
+        console.log(" Wear T-Shert")
+    }else if(temprature< 18 && temprature>  15 ){
```
Comment (florinmaticu) [inline javascript/javascript1/week2/index.js]: No need to check `temperature < 18`. In this case is not needed.
```suggestion
    }else if( temprature>  15 ){
```

File: javascript/javascript1/week2/index.js (hunk 3 above)
Comment (basafilm) [reply javascript/javascript1/week2/index.js]: It shows Sunday

Comment (basafilm) [reply javascript/javascript1/week2/index.js]: but i don't know why ? It should be -1 ?

**Pull request merged**